README.md

```diff
@@ -75,6 +75,27 @@ Run the specs:
 $ ./vendor/bin/phpspec run
 ```
 
+### Python
+
+```
+$ cd python
+```
+
+Install, create and activate the virtualenv:
+
+```
+$ pip3 install virtualenv
+$ virtualenv -p python3 venv
+$ . venv/bin/activate
+$ pip install -r requirements.txt
+```
+
+Run the tests:
+
+```
+$ venv/bin/pytest -vvv
```

Comment:
`./venv/bin/pytest` (or without the `chmod` we would need to prefix it with `python3` probably)

Reply:
but my `pytest` isn't available :thinking:

Reply:
it works globally however, `$ pytest` not sure how that works

Reply:
also `python3 -m pytest` works